dev-data/index.ts

```diff
@@ -55,6 +55,7 @@ export { mentorshipCourses, mentorshipCoursesDefault } from './mentorship.data';
 export { nodejs } from './nodejs.data';
 export { picturesSocialMediaLinks } from './pictures.data';
 export { preSchoolEn, preSchoolRu } from './preSchool.data';
+export { principleCards } from './principle-cards.data.tsx';
```

Comment:
```suggestion
export { principleCards } from './principle-cards.data';
```
maybe you don't need to use the file extension here either

Reply:
fixed